config/android/baseline.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<issues format="6" by="lint 8.1.2" type="baseline" client="gradle" dependencies="false" name="AGP (8.1.2)" variant="all" version="8.1.2">
+
+    <issue
+        id="InvalidPackage"
+        message="Invalid package reference in library; not included in Android: `java.lang.management`. Referenced from `org.jacoco.agent.rt.internal_b6258fc.JmxRegistration`.">
+        <location
+            file="$GRADLE_USER_HOME/caches/modules-2/files-2.1/org.jacoco/org.jacoco.agent/0.8.8/819fa4951ab75fba23be23f9f2793ced7fc423d7/org.jacoco.agent-0.8.8-runtime.jar"/>
+    </issue>
+
+    <issue
+        id="InvalidPackage"
+        message="Invalid package reference in library; not included in Android: `javax.management`. Referenced from `org.jacoco.agent.rt.internal_b6258fc.JmxRegistration`.">
+        <location
+            file="$GRADLE_USER_HOME/caches/modules-2/files-2.1/org.jacoco/org.jacoco.agent/0.8.8/819fa4951ab75fba23be23f9f2793ced7fc423d7/org.jacoco.agent-0.8.8-runtime.jar"/>
```

Comment:
I'm a little hesitant about the use of the hashes in those paths, in case local dev doesn't match between machiens. Are we able to use some regex to make the locations a bit more flexible?

Reply:
removed this content, auto generated by an initial ktlint on the features module I believe